gateway-messages/src/sp_impl.rs

```diff
@@ -656,7 +683,8 @@ fn handle_mgs_request<H: SpHandler>(
     let trailing_data = match &kind {
         MgsRequest::UpdateChunk(_)
         | MgsRequest::SerialConsoleWrite { .. }
-        | MgsRequest::SetIpccKeyLookupValue { .. } => leftover,
+        | MgsRequest::SetIpccKeyLookupValue { .. }
+        | MgsRequest::ResetComponentTrigger { .. } => leftover,
```

Comment:
What trailing data do we expect in `ResetComponentTrigger`

Reply:
Good catch! I removed it.